odkx-src/basics-install.rst

```diff
@@ -30,12 +30,13 @@ We also recommend installing both ODK-X Survey and ODK-X Tables. Having both is
 Installing Services
 --------------------------------
 
-  1. From your device's :guilabel:`Settings`, choose :menuselection:`Security`.
+  1. From your device's :guilabel:`Settings`, choose :menuselection:`Apps & notifications`.
 
-    - Make sure *Unknown Sources* is checked.
-    - (On older versions of Android, this setting is in :menuselection:`Applications` rather than :menuselection:`Security`)
+    - Go to :guilabel:`Special app access` in :guilabel:`Advanced` and choose :guilabel:`Install unknown apps`
+    - From the list of applications, select a browser of your choice and check :guilabel:`Allow from this source`.
+    - (On older versions of Android, this setting maybe in :menuselection:`Applications` or :menuselection:`Security` depending upon your android version. Make sure *Unknown Sources* is checked.)
```

Comment:
Can you move the older versions line to the same line as the 'Apps & notifications' as some people may stop reading at that line when they can't find it.

Reply:
Also add a comment that older versions did not have to select the web browser to allow from this source.

Reply:
@wbrunette Thanks, I made the required changes.